easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -120,6 +120,12 @@ def configure_step(self, srcdir=None, builddir=None):
 
         if builddir is None and self.cfg.get('separate_build_dir', True):
             builddir = os.path.join(self.builddir, 'easybuild_obj')
```

Comment:
@Flamefire Why not try and ensure a unique subdir here instead?

By using `self.name`, `self.version`, and may even an additional part with random chars (look for "salt", we do this in other places)?

Reply:
Because I'm lazy and this is easier.
And also because some easyconfigs rely on this name already. See SimpleElastix, SImpleITK, VV, xtb, pyGIMLi (grep for the folder name, not checked that those use CMake)

But anyway: Some ECs may need the actual build folder name in their pre/post commands so having it deterministic is a good thing. Randomizing the name would require at least a new template for that name to be used in ECs